Source/Csla/Serialization/Mobile/Configuration/MobileFormatterOptions.cs

```diff
@@ -18,6 +18,11 @@ public class MobileFormatterOptions
   /// Gets the list of custom serializers.
   /// </summary>
   public List<ITypeMap> CustomSerializers { get; } = [];
+  
+  /// <summary>
+  /// Gets a value indicating whether to use strong name checks during serialization/deserialization process.
+  /// </summary>
+  public bool UseStrongNamesCheck { get; internal set; } = true;
```

Comment:
Shouldn't this be a private set?

Or just make the setter public and let the developer set the property during configuration (skipping the two methods)?

Reply:
I have impression that you want fluent interface for the options. At least this is seems to be the pattern. Maybe it make sense to have both imperative and fluent API?

Reply:
True, fluent is easier for the end developer. The setter should still be private right?

Also, the fluent method could accept a parameter:

```c#
  public MobileFormatterOptions StrongNamesCheck(bool check)
    => UseStrongNamesCheck = check;
```

Reply:
I prefer to have it both ways, that's most inclusive in my opinion. that way library do not force preferences on developers, and I think this is not super complicated to maintain in the long run.

Reply:
I have mixed signals. Error! Error! Operator intervention required.

Reply:
Let's go with Stefan's model, I see the value in the argument.